perun/cli_groups/utils_cli.py

```diff
@@ -77,7 +74,7 @@ def create(template_type: str, **kwargs: Any) -> None:
     Unless ``--no-edit`` is set, after the successful creation of the files, an external editor,
     which is specified by :ckey:`general.editor` configuration key.
 
-    .. _jinja: http://jinja2.pocoo.org/
+    .. _jinja: https://jinja2.pocoo.org/
```

Comment:
Same as above. Current link seems to be https://jinja.palletsprojects.com/en/latest/ .